emergence_lib/src/graphics/mod.rs

```diff
@@ -93,17 +92,19 @@ fn initialize_tilemaps(
 }
 
 /// Update entities that have a newly added/changed component which implements [`IntoSprite`] with
-/// new `bevy_ecs_tilemap` [`TileBundle`](bevy_ecs_tilemap::tiles::TileBundle) information.
+/// new `bevy_ecs_tilemap` [`TileBundle`](bevy_ecs_tilemap::tiles::TileBundle) information.\
```

Comment:
This looks like a typo

```suggestion
/// new `bevy_ecs_tilemap` [`TileBundle`](bevy_ecs_tilemap::tiles::TileBundle) information.
```